### PR TITLE
pkey argument option added to the ssh method

### PR DESCRIPTION
**Pull request by elenagerman**
pkey option was added to all ssh related methods to be able define new key value param to paramiko when needed. Required for running automation for OCP Edge project, related to the paramiko issue with certificate formats when FIPS is enabled. As far as original file not suitable in this case we use PR #1643: https://github.com/paramiko/paramiko/pull/1643.
This change adds support for the rsa-sha2-256 signature algorithm specified in RFC 8332.

### CODE REVIEW

**Comment by elenagerman** [comment]
@bradmwilliams @jupierce 
please review,
Many thanks in advance!!

**Comment by elenagerman** [comment]
pkey argument was replaced with **paramiko_extra, to be more flexible

File: packages/openshift/base_verbs.py [hunk 1]
```diff
@@ -1370,6 +1370,7 @@ def node_ssh_client(apiobj_node_name_or_qname=None,
                     connect_timeout=600,
                     through_client_host=True,
                     address_type_pref="ExternalDNS,ExternalIP,Hostname",
+                    **paramiko_extra
```
Comment (jupierce) [inline packages/openshift/base_verbs.py:1373]:
```suggestion
                    paramiko_extra=None
```

File: packages/openshift/base_verbs.py [hunk 2]
```diff
@@ -1387,6 +1388,7 @@ def node_ssh_client(apiobj_node_name_or_qname=None,
     unless overridden.
     :param address_type_pref: Comma delimited list of node address types. Types will be tried in
             the order specified.
+    :param kwargs:
```
Comment (jupierce) [inline packages/openshift/base_verbs.py:1391]:
```suggestion
    :param paramiko_extra: An optional dictionary of kwargs to pass to the unlikely SSH client connection method.
```

File: packages/openshift/base_verbs.py [hunk 3]
```diff
@@ -1457,7 +1459,8 @@ def node_ssh_await(apiobj_node_name_or_qname=None,
                    key_filename=None,
                    auto_add_host=True,
                    through_client_host=True,
-                   address_type_pref="ExternalDNS,ExternalIP,Hostname"):
+                   address_type_pref="ExternalDNS,ExternalIP,Hostname",
+                   **paramiko_extra):
```
Comment (jupierce) [inline packages/openshift/base_verbs.py:1463]:
```suggestion
                   paramiko_extra=None):
```

**Pull request closed without merging**